Handler/Image.hs

```diff
@@ -1,14 +1,17 @@
-{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE CPP, RecordWildCards #-}
 
 module Handler.Image where
 
+#if MIN_VERSION_base(4,8,0)
+import Import hiding (foldMap)
```

Comment:
Also, with `-f-dev` (i.e. with `-fdev` turned **off**), the build will fail because of the warning. This is due to Bryan's insistence on having `-Werror` in the `ghc-options`.